nss-day-cohort-29/reactive-nutshell-copperden
# Set conditionals in chat and added Logout link

# Description
1. Added conditionals to the chat messages for current user vs other users: the "Edit" link will only show for the current logged in user and the messages have been styled into "bubbles" with alternating layout and different colors.
2. Added "Logout" link to the nav bar, which clears session storage and sends user to the login page. 

## Type of change

# Testing Instructions for Change Made
1. `git fetch --all`
2. `git checkout meg-chat-conditional`
3.  Click on "Messages"
4. Verify that the "bubbles" are styled differently for the current user `userName` vs other users.
5. Verify that the "Edit" link only shows for the current user `userName`.
6. With the web dev tools "Application" tab open, go to /register and create a new user. 
7. Click on "Logout" link and verify that the session storage is cleared upon clicking.

# Checklist:
